enterprise/frontend/src/metabase-enterprise/metabot/components/MetabotChat/MetabotChat.module.css

```diff
@@ -135,10 +135,109 @@
   pointer-events: auto;
   margin-bottom: 0.25rem;
   position: relative;
+  max-width: 90%;
+  align-items: flex-start;
 }
 
 .responseMessage {
   flex-grow: 1;
+  line-height: 1.5;
+}
+
+/* Markdown specific styles */
+.responseMessage p {
```

Comment:
This would be one more place where we have a slightly different markdown rendering style. Since the default built-in `<Markdown />` styling does, I'd try to at least unify this component and `enterprise/frontend/src/metabase-enterprise/ai-entity-analysis/components/AIAnalysisContent/AIAnalysisContent.tsx` by creating a `<MetabotMarkdown />` component used both here and in AIAnalysisContent. Would you need some help?